### ISSUE TITLE
Sparky.src capture bug

### ISSUE THREAD

**Issue by paxperscientiam** [fuse-box/fuse-box#1554]
Hello, I may have discovered a bug pertaining to Sparky and its ability to capture files with various extensions per [this](https://fuse-box.org/docs/task-runner/sparky#src)

The scenario is such that there are images of differing formats under "./src/assets". For the sake of this post, let's say there are just PNGs and JPGs.

I am finding that this code does NOT work ...
```ts
task("copy:images", async () => {
    await src("./**/*.(jpg|png)", { base: "./src/assets" })
        .dest("./dist/")
        .exec()
})
```
... while this code DOES work and ...
```ts
task("copy:images", async () => {
    await src("./**/*.jpg", { base: "./src/assets" })
        .dest("./dist/")
        .exec()
})
```
.. this code works too:
```ts
task("copy:images", async () => {
    await src("./**/*.png", { base: "./src/assets" })
        .dest("./dist/")
        .exec()
})
```

Basically, what happens is that Sparky is only copying files when a single extension is specified; it copies nothing when both extensions are specified.

environment:
* OS: macos 10.14.5
* fuse-box version 3.7.1
* node: 12.4.0

**Comment by nchanged** [comment]
@paxperscientiam Hi ;-) Again, you should try `v4` with the latest chokidar. I believe all those problems are solved there.

**Comment by vegarringdal** [comment]
@paxperscientiam 
FYI, base go into  dest as 2 argument in version 4
Like this:
> .dest("./dist/", "./src/assets")

Just incase you did not know about this 😄 
It tricked me earlier.
